src/locale/locales/it/messages.po

```diff
@@ -8541,23 +8541,23 @@ msgstr "Per iscriverti devi avere almeno 13 anni."
 
 #: src/components/StarterPack/ProfileStarterPacks.tsx:306
 msgid "You must be following at least seven other people to generate a starter pack."
-msgstr ""
+msgstr "Devi seguire almeno altre 7 utenti per creare uno starter pack"
```

Comment:
```suggestion
msgstr "Devi seguire almeno altre 7 utenti per creare uno starter pack."
```

Reply:
Hello! Thank you so much for the suggestions! I should've pushed a commit that should've fixed most of them. As for "Risposta a un post bloccato", well, it's actually "Rispondi a un post bloccato", so I didn't fix those lines. Thanks again!